core-services/prow/02_config/_plugins.yaml

```diff
@@ -1947,6 +1962,27 @@ plugins:
   - wip
   - yuks
   - approve
+  integr8ly/rhmi-utils:
+  - assign
+  - cat
+  - dog
+  - heart
+  - golint
+  - help
+  - hold
+  - label
+  - lgtm
+  - lifecycle
+  - override
+  - shrug
+  - sigmention
+  - size
```

Comment:
One more thing: please remove `size` too, we recently disabled it everywhere because of GH API token consumption.